qiskit_ibm_provider/ibm_backend.py

```diff
@@ -371,7 +373,7 @@ def run(
         live_data_enabled: Optional[bool] = None,
         noise_model: Optional[Any] = None,
         **run_config: Dict,
-    ) -> IBMJob:
+    ) -> RuntimeJob:
```

Comment:
We need to keep the original interface, wrapping around the runtime code, so we should not change the output type here.